config/webpack/loaders.js

```diff
@@ -5,6 +5,14 @@
 
 'use strict';
 
+var es6Loader;
+
+if (process.env.JASMINE) {
+    es6Loader = 'babel?plugins[]=rewire&plugins[]=object-assign';
```

Comment:
Did you try to run  some tests that use `babel-plugin-rewire` just in case? To make sure it still works?